endpoint/labels.go

```diff
@@ -38,6 +38,9 @@ const (
 	// AWSSDDescriptionLabel label responsible for storing raw owner/resource combination information in the Labels
 	// supposed to be inserted by AWS SD Provider, and parsed into OwnerLabelKey and ResourceLabelKey key by AWS SD Registry
 	AWSSDDescriptionLabel = "aws-sd-description"
+
+	// DualstackLabelKey is the name of the label that identifies dualstack endpoints
+	DualstackLabelKey = "dualstack"
```

Comment:
Please also add documentation for users, how it works and how use it:
you can add a file here https://github.com/kubernetes-incubator/external-dns/tree/master/docs/tutorials

Reply:
will do
and will also add some tests